current/en-us/3. fundamentals/2. creating-components.md

```diff
@@ -89,18 +89,31 @@ export class CustomerDetail {
 
 ## The Component Lifecycle
 
-All components have a well-defined lifecycle. Below is a list of methods you can implement on your view-model in order to hook into the component lifecycle:
+Every component instance has a lifecycle that you can tap into. This makes it easy for you to perform various actions at particular times. For example, you may want to execute some code as soon as your component properties are bound, but before the component is first rendered. Or, you may want to run some code to manipulate the DOM as soon as possible after your element is attached to the document. Below is a summary of the various lifecycle callbacks that you can hook into in your component.
+
+| Lifecycle Callback | Description |
+| :--- | :--- |
+| `constructor` | When the framework instantiates a component, it calls your class's constructor, just like any JavaScript class. This is the best place to put basic initialization code that is not dependent on bindable properties. |
+| `created` | The "created" hook runs just after the constructor and can be treated very similarly. The only difference is that the component's `Controller` has been instantiated and is accessible through the `$controller` property, for advanced scenarios. |
+| `bind`    | If your component has a method named "bind", then the framework will invoke it when it has begun binding values to your bindable properties. In terms of the component hierarchy, the bind hooks execute top-down, from parent to child, so your bindables will have their values set by the owning components, but the bindings in your view are not yet set. This is a good place to perform any work or make changes to anything that your view would depend on because data still flows down synchronously. This is the best time to do anything that might affect children as well. |
```

Comment:
Need @bigopon to verify the timing of the `bind` hook. It's different between au1 and au2, so some of these details don't apply here. @bigopon has worked with the au1 code around this area most recently I believe. If I recall, the `bind` hook sort of takes place in the middle of the binding process. I don't remember if the bindings in the view are set up or not actually.